js/components/reuse/card-list.vue

```diff
@@ -117,7 +117,7 @@ export default {
     props: {
         title: {
             type: String,
-            default: () => this._('Reuses')
+            default() {return this._('Reuses')}
```

Comment:
Not clear why switching back to ES5 is fixing the post view.
Noob question: it **must** be a function or would an assignment be ok (`default: this._('Reuses')`)?